code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -713,7 +734,12 @@
 		/obj/item/melee/baton/loaded, // secondary weapon, for things immune to burn, immune to ranged weapons, or for arresting low-grade threats
 		/obj/item/restraints/handcuffs/cable/zipties/cyborg,
 		/obj/item/pickaxe/drill/jackhammer, // for breaking walls to execute flanking moves
-		/obj/item/borg/destroyer/mobility
+		/obj/item/borg/destroyer/mobility,
+		/obj/item/gripper/nuclear,
+		/obj/item/crowbar/cyborg,
+		/obj/item/gps/cyborg,
+		/obj/item/pinpointer,
+		/obj/item/pinpointer/operative/nad,
```

Comment:
This'll also lead to a syndicate operative (nukie), I don't think this borg is used outside of amemery but I'm not sure if it's a good idea for them to have it